docker/trends-earth-test-pre-scripts.sh

```diff
@@ -6,5 +6,5 @@ qgis_setup.sh LDMP
 rm -rf  /root/.local/share/QGIS/QGIS3/profiles/default/python/plugins/trends.earth
 ln -sf /tests_directory/LDMP /root/.local/share/QGIS/QGIS3/profiles/default/python/plugins/trends.earth
 
-pip3 install -r /tests_directory/requirements.txt
```

Comment:
Should be using this line, not the next (or better yet call the invoke task to setup the plugin dependencies directly), so that we test with the correct packages shipped with the plugin.